### PR TITLE
Phsen pco2w sami bits updates

### PR DESCRIPTION
**Pull request by sauuyer**
PCO2W and PHSEN CC_sami_bits values updated to 14 where units have been upgraded to 14-bit Rev K boards. Board updates are captured in this Google Sheet: https://docs.google.com/spreadsheets/d/1u74ZBrWE5pZw9_auFhBxFku91GOYXml9geBjmnQ8cLw/edit#gid=1082765693

### CODE REVIEW

**Review by snwhite — APPROVED** [review]
I checked these against the Instrument Refurb spreadsheet and they all look correct to me.

File: calibration/PCO2WC/CGINS-PCO2WC-C0107__20211228.csv [hunk 1]
```diff
@@ -7,5 +7,5 @@ C0107,CC_ea434,19706,constant
 C0107,CC_ea620,34,constant
 C0107,CC_eb434,3073,constant
 C0107,CC_eb620,44327,constant
-C0107,CC_sami_bits,12,
+C0107,CC_sami_bits,14,
```
Comment (rtravis-whoi) [inline calibration/PCO2WC/CGINS-PCO2WC-C0107__20211228.csv:10]: good

File: calibration/PCO2WC/CGINS-PCO2WC-C0108__20220103.csv [hunk 2]
```diff
@@ -7,5 +7,5 @@ C0108,CC_ea434,19706,constant
 C0108,CC_ea620,34,constant
 C0108,CC_eb434,3073,constant
 C0108,CC_eb620,44327,constant
-C0108,CC_sami_bits,12,
+C0108,CC_sami_bits,14,
```
Comment (rtravis-whoi) [inline calibration/PCO2WC/CGINS-PCO2WC-C0108__20220103.csv:10]: good

File: calibration/PCO2WC/CGINS-PCO2WC-C0120__20211214.csv [hunk 3]
```diff
@@ -7,5 +7,5 @@ C0120,CC_ea434,19706,constant
 C0120,CC_ea620,34,constant
 C0120,CC_eb434,3073,constant
 C0120,CC_eb620,44327,constant
-C0120,CC_sami_bits,12,
+C0120,CC_sami_bits,14,
```
Comment (rtravis-whoi) [inline calibration/PCO2WC/CGINS-PCO2WC-C0120__20211214.csv:10]: good

File: calibration/PCO2WC/CGINS-PCO2WC-C0140__20220131.csv [hunk 4]
```diff
@@ -7,5 +7,5 @@ C0140,CC_ea434,19706,constant
 C0140,CC_ea620,34,constant
 C0140,CC_eb434,3073,constant
 C0140,CC_eb620,44327,constant
-C0140,CC_sami_bits,12,
+C0140,CC_sami_bits,14,
```
Comment (rtravis-whoi) [inline calibration/PCO2WC/CGINS-PCO2WC-C0140__20220131.csv:10]: good

File: calibration/PHSENE/CGINS-PHSENE-P0177__20220127.csv [hunk 5]
```diff
@@ -6,4 +6,4 @@ P0177,CC_eb578,38676.0,
 P0177,CC_ind_off,0,
 P0177,CC_ind_slp,1,
 P0177,CC_psal,35,Constant
-P0177,CC_sami_bits,12,
+P0177,CC_sami_bits,14,
```
Comment (rtravis-whoi) [inline calibration/PHSENE/CGINS-PHSENE-P0177__20220127.csv:9]: good

File: calibration/PHSENE/CGINS-PHSENE-P0178__20220127.csv [hunk 6]
```diff
@@ -6,4 +6,4 @@ P0178,CC_eb578,38676.0,
 P0178,CC_ind_off,0,
 P0178,CC_ind_slp,1,
 P0178,CC_psal,35,Constant
-P0178,CC_sami_bits,12,
+P0178,CC_sami_bits,14,
```
Comment (rtravis-whoi) [inline calibration/PHSENE/CGINS-PHSENE-P0178__20220127.csv:9]: good

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**